x/staking/common_test.go

```diff
@@ -41,13 +37,15 @@ func getBaseSimappWithCustomKeeper(t *testing.T) (*codec.LegacyAmino, *simapp.Si
 	ctx := app.BaseApp.NewContext(false, tmproto.Header{})
 
 	appCodec := app.AppCodec()
-
+	stakingConfig := types.DefaultConfig()
+	stakingConfig.PowerReduction = sdk.NewIntFromBigInt(new(big.Int).Exp(big.NewInt(10), big.NewInt(18), nil))
 	app.StakingKeeper = keeper.NewKeeper(
```

Comment:
There might be a bug with this: in L36, we setup simapp, which creates a staking keeper with the default power reduction factor (and that calls InitGenesis already). Then in L42, we overwrite with a new keeper. So the power reduction factor hot changes during the app.